script/Ciel/Day 3/03_01_CIEL03_3_2.txt

```diff
@@ -234,7 +234,7 @@ Even now, I still don't understand the reason.
 -- Page 122, Offset 5144.
 -- でも、自分の事なんて、どうでもいい。
 // As in, Shiki is saying that he's not important, he shouldn't think about himself but rather about the girl he killed
-But who cares about me?
+But who cares about me? // "But enough about me" to fit comment? or "Whatever happens to me doesn't matter"?
```

Comment:
Ehh 'enough about me' kinda strikes me as too joking / something you'd say during an awkward introduction to your sitcom.